api/desecapi/dns.py

```diff
@@ -30,7 +30,7 @@ def from_text(cls, rdclass, rdtype, tok, origin=None, relativize=True):
             raise dns.exception.UnexpectedEnd
         return cls(rdclass, rdtype, strings)
 
-    def to_wire(self, file, compress=None, origin=None):
+    def _to_wire(self, file, compress=None, origin=None, canonicalize=False):
```

Comment:
dead code (and `canonicalize` argument is also dead)

Reply:
very much alive :) change due to dnspython 2.0